### ISSUE TITLE
ARIA form features

### ISSUE THREAD

**Issue by francisrupert** [USDA-FSA/fsa-style#222]
Some are missing, or need more explicit extensions.

* `aria-required="true"`
* `aria-describedby=[id_value]` (especially on `.fsa-field__help`)
* `aria-invalid="true"`
* `role="alert"`

**Comment by francisrupert** [comment]
dupe of #156
